modules/configuring-a-workload-to-use-shared-cpus.adoc

```diff
@@ -0,0 +1,132 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="configuring-a-workload-to-use-shared-cpus_{context}"]
+= Configuring a workload to use shared CPUs
+
+You can pin lightweight workload tasks to shared CPUs to improve CPU resource efficiency. When you enable shared CPUs on a node, containers deployed on the node feature the `OPENSHIFT_SHARED_CPUS` and `OPENSHIFT_ISOLATED_CPUS` environment variables. These variables show the IDs of the shared and isolated CPUs. 
```

Comment:
I'm failing to understand this: "When you enable shared CPUs on a node, containers deployed on the node feature the `OPENSHIFT_SHARED_CPUS` and `OPENSHIFT_ISOLATED_CPUS` environment variables"